velox/docs/develop/scalar-functions.rst

```diff
@@ -485,6 +485,34 @@ For example, to specify rand signature with a constant seed argument:
 
     registerFunction<RandFunction, double, Constant<int32_t>>({"rand"});
 
+Function Name Prefixes
```

Comment:
I feel this is very Presto-specific. How about we move it to https://facebookincubator.github.io/velox/functions.html and clarify that these instructions are for use case with Presto engine specifically. Is there a corresponding documentation in Presto we could reference as well?